lib/src/front_end/piece_factory.dart

```diff
@@ -542,6 +532,17 @@ mixin PieceFactory {
     return builder.build();
   }
 
+  /// Creates an [AdjacentPiece] for a given record type field.
+  Piece createRecordTypeField(RecordTypeAnnotationField node) {
+    // TODO(tall): Format metadata.
```

Comment:
sounds like something that is already done generally in this repo, but what is `TODO(tall)`? I was expecting a `TODO(your-ldap-here)` instead.

Reply:
We are essentially writing an entirely new formatter to support the new style (while still supporting the old one in the interim). So in the new formatter code, like PieceFactory here, no existing language feature is supported until we implement it again. Metadata isn't implemented yet.

We're using `TODO(tall)` as marker comments to keep track of work left to do in the new implementation. The idea is that the new implementation isn't done until all of those (and `throw UnimplementedError` and `TODO(perf)`) are addressed and removed.

Reply:
totally makes sense, thanks for clarifying!

Long ago we had a `TODO(redemption)` in dart2js for the same purpose... to my surprise, we still have 6 of those left :open_mouth: